### ISSUE TITLE
Replace custom jacoco report aggregation with Gradle's one

### ISSUE THREAD

**Issue by snazy** [projectnessie/nessie#5991]
See https://docs.gradle.org/7.4/samples/sample_jvm_multi_project_with_code_coverage_distribution.html and https://docs.gradle.org/7.4/userguide/jacoco_report_aggregation_plugin.html

Hints:
* Replace usage of `org.projectnessie.buildsupport.jacoco` plugin in `nessie-conventions.gradle.kts` with `jacoco`
* Update `:nessie-code-coverage` project, remove `org.projectnessie.buildsupport.jacoco-aggregator` plugin
* Apply similar changes to projectnessie/cel-java
* Remove the jacoco plugins from projectnessie/gradle-build-plugins

**Comment by snazy** [comment]
TL;DR Gave this a shot, but the experience is bad.

To use Gradle's Jacoco-aggregation, one has to use the JvmTestSuite plugin as well, which means that the current `intTest` has to be refactored to use that plugin. There are a couple of issues w/ the JvmTestSuite plugin:
* The docs say, that it's easy to add all "main" dependencies - but that does not work for dependency declarations that use `platform(...)` (including bom's)
* Sharing sources between the `test` and `intTest` test suites is also tricky.

In other words: a lot of effort for no big win.
